scripts/Invoke-SasWorkstationProvisioner.ps1

```
@@ -0,0 +1,218 @@
+[CmdletBinding(SupportsShouldProcess = $true, ConfirmImpact = 'High')]
+param(
+    [ValidateSet('Audit', 'Plan', 'Apply', 'Rollback')]
+    [string]$Action = 'Plan',
+    [string]$AgentSwitchboardRef = 'feat/windows-profile-open-or-activate',
+    [string]$AgentSwitchboardRepo = 'EndeavorEverlasting/AgentSwitchboard',
+    [string]$LauncherRelativePath = 'tooling/profiles/windows/Invoke-AgentSwitchboardOpenOrActivate.ps1',
+    [string]$ManifestRelativePath = 'tooling/profiles/windows/tmux-new-instance-shortcut.example.json',
+    [string]$InstallRoot = (Join-Path $env:LOCALAPPDATA 'AgentSwitchboard\profiles\windows'),
+    [string]$StateRoot = (Join-Path $env:LOCALAPPDATA 'SysAdminSuite\workstation'),
+    [string]$WslDistribution = 'Ubuntu',
+    [int]$TimeoutSeconds = 30
```

PSScriptAnalyzer (1.25.0)

[warning] The parameter 'TimeoutSeconds' has been declared but not used.

(PSReviewUnusedParameter)